python/doc/examples/calibration/bayesian_calibration/plot_gibbs_simus.py

```diff
@@ -0,0 +1,598 @@
+mean_true = np.dot( X, theta_true ).ravel()
+std_true = np.sqrt(1. / tau_true + 1. / Q).ravel()
+
+Y_sim = mean_true + [x[0] for x in ot.Normal().getSample(n)] * std_true
+Yobs_sim = np.round(Y_sim/delta)*delta
+
+
+# %%
+# Plot the simulated dataset.
+plt.plot(X[:,1], Yobs_sim, '.')
+plt.plot(X[:,1], mean_true)
+plt.legend(("Simulations", "Simulation mean"))
+plt.xlabel("X", fontsize=16)
+plt.ylabel("$Y^{obs}$", fontsize=16)
+plt.title("Simulated data")
+
+# %%
+# 2. Bayesian Inference
+# ---------------------
+# 
+# 2.1. Choice of a prior law
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# 
+# We use the standard Normal-Gamma prior for :math:`\theta` et
+# :math:`\tau`:
+# 
+# .. math::
+# 
+#    \theta | \tau \sim \mathcal N_{p}(\mu_0; \tau^{-1}\Sigma_0) \\
+#    \tau \sim \mathcal G(a_0, b_0),
+# 
+# where all parameters are assumed *a priori* independent if not stated
+# otherwise.
+# 
+# Furthermore, a default choice for the hyperparameters consists in having
+# all prior variances go to infinity, equivalent to the degenerate case:
+# 
+# .. math::
+# 
+#    \mu_0, \Sigma_0^{-1}, a_0, b_0 \equiv 0
+# 
+# But the resulting prior is improper. Hence, posterior propriety needs to
+# be proven first.
+# 
+# A simpler solution is to ensure prior (hence posterior) propriety by
+# imposing bounds :math:`\theta_{\min}, \theta_{\max}, \tau_{\min}, \tau_{\max}`
+# on all parameters following:
+# 
+# .. math::
+# 
+#    \theta | \tau \sim \mathcal N_{p}(\mu_0; \tau^{-1}\Sigma_0) {\bf 1}_{\{\theta_{\min} \leq \theta \leq \theta_{\max}\}} ;\\
+#    \tau \sim \mathcal G(a_0, b_0){\bf 1}_{\{\tau_{\min} \leq \tau \leq \tau_{\max}\}},
+# 
+# where inequalites are taken componentwise. When all hyperparameters go to
+# :math:`0` as described above, the prior converges to a product of
+# uniform distributions.
+# 
+# We will use this product of univariate uniforms as a prior in the
+# following. As discussed above, there is no simple way to obtain the
+# posterior distribution, justifying the use of Monte-Carlo Markov chain
+# techniques, as described hereafter.
+
+# %%
+# Upper/lower bounds on uncertain variables 
+
+lower = (Yobs_sim.ravel()-delta).tolist()
+upper = (Yobs_sim.ravel()+delta).tolist()
+
+support = ot.Interval([-2] * p + [1e-4] + lower, [2] * p + [1e1] + upper)
+
+# Initialize to true value
+initial_state = theta_true[:,0].tolist() + [tau_true] + Y_sim.ravel().tolist()
+initial_state = ot.Point(initial_state)
+
+# %%
+# 2.2. Posterior sampling
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# 
+# The solution we advocate consists in sampling from the joint posterior
+# distribution of all uncertain parameters, including the vector
+# continuous intensities :math:`Y,` seen here as a latent variable.
+# Indeed, adding :math:`Y` to the vector of sampled variables yields a
+# posterior density which is available in closed form, up to an unkown
+# multiplicative factor
+# 
+# .. math::
+# 
+#    \pi(Y,\theta,\tau|Y^{obs})
+#    \propto
```

Comment:
Please make the `propto` symbols aligned, to make the equations more readable:
![image](https://user-images.githubusercontent.com/31351465/160208140-a2723ff2-3b4d-4a61-9847-7e335d134f41.png)